HtmlMesh/src/html-mesh.ts

```diff
@@ -166,7 +278,8 @@ export class HtmlMesh extends Mesh {
         this.scaling.setAll(1);
 
         if (element) {
-            this._element!.appendChild(element);
+            const sizeFitHandler = SizeFitHandlerMap[this._sizeFit]
```

Comment:
This all gets replaced with `this._element!.appendChild(this._fitStrategy.wrapElement(element)`